packages/react-router-dom/modules/__tests__/BrowserRouter-test.js

```diff
@@ -14,7 +14,7 @@ describe("A <BrowserRouter>", () => {
 
   describe("with a `history` prop", () => {
     it("logs a warning to the console", () => {
-      spyOn(console, "error");
+      jest.spyOn(console, "error").mockImplementation(() => {});
```

Comment:
So this is more verbose - that's because by default Jasmine `spyOn` doesn't call the underlying implementation while `jest.spyOn` does (i.e. we only spy on the calls, we don't mock them). That means if we switch this over to just `jest.spyOn` then the real console.error will be called and output to the terminal

Thus, to prevent calling the real `console.error`, we provide a mock implementation for the spy 👍

Reply:
In that case, why wouldn't we want to just use the Jasmine implementation? I mean, we kinda want that behavior, right? Is using Jasmine going to cause problems for us? I don't have a strong opinion about it, but seems like it'd be simpler to just leave it as it was. Happy to defer to you if you think this is more robust long-term.

Reply:
Oh, nevermind. I just re-read [the original message](https://github.com/ReactTraining/react-router/pull/6427#issue-226229605) and realized you said we need to use `jest.spyOn` because we're using `jest-circus` now. Gotcha.